src/App.tsx

```diff
@@ -1,13 +1,16 @@
-import ErrorBoundary from 'Components/common/error-boundary';
-import Layout from 'Components/layout';
-import AuthProvider from 'Contexts/authProvider';
+import ErrorBoundary from './components/common/error-boundary';
+import Layout from './components/layout';
+import useLogin from './api/hooks/useLogin';
```

Comment:
```suggestion
import useLogin from 'Api/hooks/useLogin';
```

Can we add this to our vite and ts config, so we can import it like this